swapi/src/scripts/dataModel.js

```diff
@@ -0,0 +1,39 @@
+class Model {
+    constructor() {
+		this.resultVolume = 5;
+    }
+
+    async getApiData(addUrl) {
+    	let apiData = await fetch(`https://swapi.dev/api/${addUrl}`)
+			.then((response) => 
+				response.json()
+			)
+			.catch(()=> {
+				alert("Error. Try refresh page.")
```

Comment:
please don't use alert for errors. it's annoying, use `console.error`

Reply:
I fixed this.
```suggestion
.catch((error) => {
        console.error(error);
      });
```